github/client.go

```diff
@@ -134,6 +134,23 @@ func (client *Client) CreatePullRequest(project *Project, params map[string]inte
 	return
 }
 
+func (client *Client) GetPullRequestUrlsForHead(headProject *Project, branch string) (pr []PullRequest, err error) {
+	api, err := client.simpleApi()
+	if err != nil {
+		return
+	}
+
+	res, err := api.Get(fmt.Sprintf("repos/%s/%s/pulls?head=%s:%s", headProject.Owner, headProject.Name, headProject.Owner, branch))
```

Comment:
I tried using "base" initially, but I couldn't get a valid response, I even contacted github support. Their response was:

> You'll want to specify the head branch, not the base branch, like this:
> https://api.github.com/repos/fsharp/FAKE/pulls?head=fsharp:FSI
> The head is the source of the pull request (the branch from which the pull request is made), and the base is the target of the pull request (the branch to which the pull request is made).

Hope this makes sense...